relay-server/src/actors/envelopes.rs

```diff
@@ -1022,6 +1038,23 @@ impl EnvelopeProcessor {
         Ok(())
     }
 
+    /// Expands profile items inside an envelope
+    #[cfg(feature = "processing")]
+    fn expand_profile(&self, state: &mut ProcessEnvelopeState) -> Result<(), ProcessingError> {
+        let envelope = &mut state.envelope;
+        let meta = envelope.meta();
+
+        if meta.client() != Some(utils::ANDROID_SDK_NAME) {
```

Comment:
I'm still looking for the right way to identify an Android client in Relay. Is this correct?